pr2_pbd_interaction/HeadPoseData head_pose
---
bool status